serving/samples/telemetry-go/README.md

```diff
@@ -89,7 +94,8 @@ Inspect the created resources with the `kubectl` commands:
 
 To access this service via `curl`, you need to determine its ingress address.
 
-1. To determine if your service is ready:
+1. To determine if your service is ready:  
+Check the status of your Knative gateway:
```

Comment:
I think you want to indent this and the next block to have the ordered list work correctly.